### DISCUSSION TITLE
Decimal point for some locales

### DISCUSSION THREAD

**Discussion by kvn61** [LMS-Community/slimserver#900] [General]
May be a good idea add to logitechmediaserver.service line:
Environment="LC_NUMERIC=C"
I have decimal point as ',' and after migrating from win to linux two day have battle with transcoding rules:
ERROR: invalid value for --skip

Thanks for best media server.

**Comment by michaelherger** [comment]
This should "just work". What locale are you using in general? What language settings on Windows? Your files should match the same.
